src/test/java/unit/CromwellConfig.java

```diff
@@ -0,0 +1,59 @@
+package unit;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import harness.TestCommand;
+import harness.TestUser;
+import harness.baseclasses.SingleWorkspaceUnit;
+import harness.utils.WorkspaceUtils;
+import java.io.File;
+import java.io.IOException;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+/** Tests for the `terra cromwell` commands. */
+@Tag("unit")
+public class CromwellConfig extends SingleWorkspaceUnit {
+  @Test
+  @DisplayName("cromwell config create in root path")
+  void cromwellConfigCreateRoot() throws IOException {
+    TestUser testUser = TestUser.chooseTestUserWithSpendAccess();
+    testUser.login();
+    WorkspaceUtils.createWorkspace(testUser);
```

Comment:
You don't need to create a workspace. `SingleWorkspaceUnit` creates a workspace. It's called SingleWorkspaceUnit because it creates a single workspace, and it's used for unit tests.